api/endpoint/tickers.go

```diff
@@ -88,8 +88,12 @@ func PostTickersHandlerV2(controller controllers.TickersController) func(c *gin.
 			return
 		}
 		request.Ids = removeDuplicates(request.Ids)
+		currency := request.Currency
+		if len(currency) == 0 {
```

Comment:
add test?

Reply:
In Postman.

Reply:
I would suggest to keep such test cases inside codebase, it's external tool

Reply:
https://github.com/trustwallet/watchmarket/issues/397